spring-cloud-gcp-pubsub-stream-binder/src/main/java/com/google/cloud/spring/stream/binder/pubsub/provisioning/PubSubChannelProvisioner.java

```diff
@@ -79,7 +80,14 @@ public ConsumerDestination provisionConsumerDestination(String topicName, String
 
 		String subscriptionName;
 		Subscription subscription;
-		if (StringUtils.hasText(group)) {
+		if (StringUtils.hasText(properties.getExtension().getCustomSubscription())) {
+			String customName = properties.getExtension().getCustomSubscription();
```

Comment:
Might as well move this line to before `if` block, since you need to check its contents anyway.

Reply:
Good catch - done.